testkit/src/main/scala/org/bitcoins/rpc/BitcoindRpcTestUtil.scala

```diff
@@ -122,12 +136,28 @@ trait BitcoindRpcTestUtil extends BitcoinSLogger {
     } else firstAttempt
   }
 
-  def startServers(servers: Vector[BitcoindRpcClient])(
+  /**
+    * Starts the given servers, and waits for them (in a blocking manner)
+    * until they are started.
+    */
+  def startServers(servers: BitcoindRpcClient*)(
       implicit system: ActorSystem): Unit = {
     servers.foreach(_.start())
     servers.foreach(RpcUtil.awaitServer(_))
   }
 
+  /**
+    * Stops the given servers, and waits for them (in a blocking manner)
+    * until they are stopped. Deletes their temp dir.
+    */
+  def stopServers(servers: BitcoindRpcClient*)(
+      implicit system: ActorSystem): Unit =
+    servers.foreach(server => {
+      server.stop()
+      RpcUtil.awaitServerShutdown(server)
```

Comment:
If you made this two separate calls (one that calls stop for each server, and the other that waits for it to stop and then deletes its TmpDir ), then this would be much faster for longer lists of servers where currently it is waiting to completely take down one server before starting on the next